src/main/scala/mission/impossibl/bots/source/WasteSource.scala

```diff
@@ -60,6 +65,10 @@ object WasteSource {
             context.log.info("Collection Timeout")
             context.self ! CheckGarbageLevel()
             source(instance, state.copy(collectionTimeout = None))
+
+          case GarbageScoreSummary(garbage_score) =>
+            context.log.info("Waste Source got its Score")
+            source(instance, state.copy(score = state.score.updated(garbage_score)))
```

Comment:
updated to metoda list, tu masz inta,  możesz zmienić na listę  i wtedy będzie się appendować

Reply:
zmieniłem sposób nadpisu po prostu

Reply:
Ciągle widzę to compilation errors związane z tym że to int a nie lista jbc

Reply:
dopiero push poszedł :D